guides/v2.1/cloud/cdn/cloud-fastly.md

```diff
@@ -10,44 +10,50 @@ functional_areas:
-
-* Create edge and ACL dictionaries with VCL snippets (Varnish 2.1 compliant) to
-  modify how caching responds to requests
-
-* Three types of purges for:
-
-  * Quick Purge (by URL)
-  * Surrogate/key purge using tags to purge specific HTML, images, categories, and so on
-  * Purge all, which clears everything in the cache
-* GeoIP extension support
-* Force unencrypted requests over to TLS
-* Image optimization support offloads image optimization to the Fastly service
-  to speed up image delivery for your store
-
-We highly recommend enabling and using Fastly for your caching and CDN. The only
+Fastly provides the following services to optimize and secure content delivery operations for your {{ site.data.var.ece }} projects. These services are included with your {{ site.data.var.ece }} account at no additional cost:  These services are included with your {{ site.data.var.ece }} account at no additional cost: 
+
+-  **Content delivery network (CDN)**—Varnish-based service that caches your site pages, assets, CSS, and more in backend data centers you set up. As customers access your site and stores, the requests hit Fastly to load cached pages faster. The CDN service provides the following features: 
+	
+   -  Caches your site pages, assets, CSS, and more in backend data centers that you set up to reduce bandwith load and costs
+	
+   -  Create [edge and ACL dictionaries with VCL snippets](#custom-vcl) (Varnish 2.1 compliant) to modify how caching responds to requests
+
+   -  Configure [purge options]({{ page.baseurl }}/http://127.0.0.1:4000/guides/v2.1/cloud/cdn/configure-fastly.html#purge) to clear generated content
+   
+   -  [GeoIP service support]({{ page.baseurl }}/cloud/cdn/configure-fastly.html#geoip)
+
+   -  [Force unencrypted requests over to TLS](#tls)
+   
+   -  [Customize Fastly timeout](#timeouts) settings to prevent 503 responses on bulk operation requests 
+	
+   -  [Custom error or maintenance page]({{ site.baseurl }}/cloud/cdn/configure-fastly.html#fastly-errpg)
+   
+-  **Security**—After you set up your {{ site.data.var.ece }} projects to use the Fastly CDN, additional security features are available to protect your sites and network.
+
+      -  [**DDoS protection**](#ddos)—Built-in protection against common attacks like Ping of Death, Smurf attacks, as well as other ICMP-based floods.
+
+      -  **[Managed Cloud WAF]({{ site.baseurl }}/cloud/cdn/fastly-waf-service.html)**—Managed web application firewall service that provides PCI-compliant protection to block malicious traffic before it can damage your production {{ site.data.var.ece }} sites and network.
+   
+-  **Image optimization**—Offloads image processing and resizing load to the Fastly service freeing servers to process orders and conversions efficiently. See [Fastly image optimization]({{ page.baseurl }}/cloud/cdn/fastly-image-optimization.html).
+
+We highly recommend enabling and using Fastly for your CDN, security, and image optimization needs. The only
```

Comment:
Using the word "want". Maybe can word it with more explanation about the situation.